ghost/admin/app/routes/posts/analytics.js

```diff
@@ -43,6 +46,20 @@ export default class Analytics extends AuthenticatedRoute {
         if (user.isContributor && !post.isDraft) {
             return this.replaceWith(returnRoute);
         }
+
+        if (this.routeName === 'posts.analytics.posts-x') {
```

Comment:
This is for preventing non-admins or sites without the config to be able to view the new Post Analytics